scripts/src/main/resources/scripts/command/gradle

```diff
@@ -34,5 +34,5 @@ if [ "${setup}" = "true" ]
 then
   doRunCommand "gradle -v" "verify installation of gradle"
 else
-  gradle ${@}
-fi
\ No newline at end of file
+  gradle "${@}"
```

Comment:
Again something similar. Maybe I am also mistaken but IMHO quoting provides everything as a single argument, or not? Then this would be a bug.

Reply:
Quoting of strings or quoting in string context turns the result into a single string. But $@ is a array variable and the standard command line is not a string context. So the result of "$@" is - for arguments ["foo","bar","foo bar"] - again:  "foo" "bar" "foo bar",

see https://wiki-dev.bash-hackers.org/scripting/posparams#mass_usage